setup.py

```diff
@@ -3,7 +3,7 @@
 
 setup(
     name='scrapy-splash',
-    version='0.7.2',
+    version='0.8.0',
```

Comment:
This is from merging the 0.8.0 branch into master. The actual version bump will be done as part of the release procedure, *after* we merge these changes.